chia/full_node/mempool_manager.py

```diff
@@ -501,7 +501,7 @@ async def validate_spend_bundle(
                 for c_sb_id in self.mempool.removals[conflicting.name()]:
                     sb: MempoolItem = self.mempool.spends[c_sb_id]
                     conflicting_pool_items[sb.name] = sb
-            log.warning(f"Conflicting pool items: {len(conflicting_pool_items)}")
+            log.info(f"Conflicting pool items: {len(conflicting_pool_items)}")
```

Comment:
I agree, this log message isn't great

Reply:
@emlowe the next line, where we call `can_replace()` will print any reasons why the replace failed as `DEBUG` and only if it succeeds will it print (as `INFO`) that some items were replaced. Which makes me thing that perhaps this should really be `DEBUG` as well. It's a bit awkward to log that a replace is attempted, but then not hear about it anymore (because it failed).

Presumably we print the failures as debug, as it could just be someone spamming the mempool.